benchmarks/benchmarks_suite_test.go

```diff
@@ -113,26 +111,12 @@ func ConfigureScenarioResultDirectories(scenarios *config.Scenarios, directory s
 		configurations = append(configurations, scenarios.HighVolumeWrites.Configurations...)
 	}
 
-	CreateResultsDirectoryFor(configurations, directory)
 	CreateResultsReadmeFor(configurations, directory)
 }
 
-func CreateResultsDirectoryFor(configurations []config.Configuration, directory string) {
-	for _, configuration := range configurations {
-		dirName := sanitize.BaseName(configuration.Description)
-		path := filepath.Join(directory, dirName)
-
-		if _, err := os.Stat(path); os.IsNotExist(err) {
-			if err := os.Mkdir(path, 0700); err != nil {
-				panic(fmt.Sprintf("Failed to create report directories, error %s", err))
-			}
-		}
-	}
-}
-
 func CreateResultsReadmeFor(configurations []config.Configuration, directory string) {
 	path := filepath.Join(directory, "README.md")
-	file, err := os.Create(path)
+	file, err := os.OpenFile(path, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600)
```

Comment:
You can probably just create the readme file in the `Makefile` after this line: https://github.com/observatorium/loki-benchmarks/blob/master/Makefile#L29.

Also, I don't think you need the `600` at the end of this.

Reply:
I need to pass the third argument (the flag argument), I can't call that OS function without it.

Reply:
BTW, in this file we can use the Create instead of OpenFile, that works as well